src/view/edit-event-view.js

```diff
@@ -57,26 +58,29 @@ const createEditEventTemplate = (event, destinations, offers) => {
   </li>`);
 };
 
-export default class EditEventView {
-  constructor({event, destinations, offers}) {
-    this.event = event;
-    this.destinations = destinations;
-    this.offers = offers;
+export default class EditEventView extends AbstractView {
+  #event = null;
+  #destinations = null;
+  #offers = null;
+  #handleViewClick = null;
+
+  constructor({event, destinations, offers, onFormSubmit}) {
```

Comment:
Придерживаемся единого стиля именования

```suggestion
  constructor({event, destinations, offers, handleViewClick}) {
```